### PR TITLE
Portflios x Axis changes

### PR DESCRIPTION
**Pull request by cazapatamar**

### CODE REVIEW

File: src/pages/search/shared_components/charts/SmallBars.tsx [hunk 1]
```diff
@@ -76,11 +78,15 @@ class SmallBars extends React.Component<Props, State> {
       maxValue = "auto",
       enableLabel = false,
       alternateAxisY = { values: {} },
+      gridXValues,
```
Comment (erikasv) [inline src/pages/search/shared_components/charts/SmallBars.tsx:81]: What's the default value for this prop?

Comment (cazapatamar) [reply src/pages/search/shared_components/charts/SmallBars.tsx:81]: That is undefined

Comment (erikasv) [reply src/pages/search/shared_components/charts/SmallBars.tsx:81]: assign it, the same way you did with `tickValues`

Comment (cazapatamar) [reply src/pages/search/shared_components/charts/SmallBars.tsx]: Fixed

**Review by erikasv — APPROVED** [review]

**Pull request merged**